src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs

```diff
@@ -23,14 +23,16 @@ namespace Tizen.NUI
 
     internal class TransitionItemBase : BaseHandle
     {
+        private bool appearingTransition = true;
+
         /// <summary>
         /// Creates an initialized TransitionItemBase.<br />
         /// </summary>
-        public TransitionItemBase(View target, bool isAppearing, TimePeriod timePeriod, AlphaFunction alphaFunction) : this(Interop.TransitionItemBase.New(), true)
+        public TransitionItemBase(View target, bool appearingTransition, TimePeriod timePeriod, AlphaFunction alphaFunction) : this(Interop.TransitionItemBase.New(), true)
```

Comment:
could you resolve the warning below?

**src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs#L31
Parameter target of method .ctor is never used. Remove the parameter or use it in the method body.**

Reply:
I added Log to use the target argument.